stable/metrics-server/templates/cluster-role.yaml

```diff
@@ -19,4 +19,13 @@ rules:
       - get
       - list
       - watch
+ {{- if not .Values.kubeletReadOnlyPort }}
```

Comment:
This doesn't have to be conditional. It will anyways be required in future releases of metrics-server (see above)